optin/reporting/test/models/runDBNotFoundTests.js

```diff
@@ -0,0 +1,23 @@
+const Config = require('../../config');
+const { refreshDBs } = require('../../models/database');
+const { expect } = require('chai');
+const { stub, restore } = require('sinon');
+const path = require('path');
+
+const runDBNotFoundTests = ({ paginationFunc, parameters }) => {
+	beforeEach(restore);
+	it('throw error if db not found', async () => {
+		try {
+			// Arrange:
+			stub(Config, 'getDataStoragePath').returns(path.join(__dirname, '../path/notFound'));
+			refreshDBs();
+
+			await paginationFunc(parameters);
+		} catch (error) {
+			// Assert:
+			expect(error.message).to.be.equal('Database error :SQLITE_CANTOPEN: unable to open database file');
+		}
+	});
```

Comment:
this test will pass if there is no exception. you can return from catch handler and have additional assert at end of function that always fails.

Reply:
it's like this way?

```js
expect(() => func())).to.throw(Error);
```